lib/models/project.js

```diff
@@ -104,6 +104,10 @@ Project.prototype.hasDependencies = function() {
  */
 Project.prototype.setupNodeModulesPath = function() {
   this.nodeModulesPath = nodeModulesPath(this.root);
+
+  this.emberCliNodeModulesPath = this.nodeModulesPath &&
```

Comment:
> @stefanpenner @rwjblue Is getting ember-cli's node_modules path like this combined with path.join(this.emberCliNodeModulesPath, 'ember-cli-legacy-blueprints') (used below in supportedInternalAddonPaths) okay?

no, we can't be sure it is installed there. we should typically use `require.resolve(module-name)` to find where on disk it is.